### PR TITLE
[Telegram API] added telegram bot integration and Notification Service

### PR DESCRIPTION
**Pull request by TetianaDenchyk**

### CODE REVIEW

File: src/main/java/com/champions/carsharingservice/controller/HealthController.java [hunk 1]
```diff
@@ -9,6 +9,6 @@
 public class HealthController {
     @GetMapping("/hello")
     public String healthCheck() {
-        return "Hello world!";
```
Comment (Oleksandr-Tymoshenko) [inline src/main/java/com/champions/carsharingservice/controller/HealthController.java:12]: why?🤣

**Review by vsych321 — APPROVED** [review]
LGTM

File: src/main/java/com/champions/carsharingservice/telegram/CarSharingBot.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    /*
+        1. You should create your bot in Telegram (I will send you a video)
+        2. Save token of the bot to env. file and after as variable in application.properties
+        3. Save name of your bot to env. file and after as variable in application.properties
+        4. Run the app and click /start in your bot
+        5. In the console you will see chat id which you should add to
+        env. file and after as variable in application.properties (just numbers)
+        6. After all members do that we will delete this instruction
+        and functionality to get chat id in method onUpdateReceive
+     */
```
Comment (Oleksandr-Tymoshenko) [inline src/main/java/com/champions/carsharingservice/telegram/CarSharingBot.java:31]: do not forget to remove

**Review by Oleksandr-Tymoshenko — APPROVED** [review]

File: src/main/java/com/champions/carsharingservice/controller/HealthController.java (hunk 1 above)
Comment (TetianaDenchyk) [reply src/main/java/com/champions/carsharingservice/controller/HealthController.java:12]: I am friendly and like talking to world 😄

**Pull request merged**